src/server.js

```diff
@@ -69,23 +70,33 @@ const initServer = async function (db, config) {
         }, 'stdout']
       }
     }
-  }])
+  }, {
 
-  // add a health endpoint on /
-  server.route({
-    method: 'GET',
-    path: '/',
-    handler: async (request, h) => {
-      if (!(await db.isConnected())) {
-        return h.response({
-          statusCode: 500,
-          error: 'Internal Server Error',
-          message: 'Database not connected' }).code(500)
+  // add a health endpoint on Database being available/
+    plugin: Health,
+    options: {
+      path: '/health', //Health route path
+      tags: ['health', 'monitor'],
+      responses: {
+        healthy: {
+          message: 'Quoting service Database connection is healthy!!!'
```

Comment:
The /health end-point is to report on the quoting-service health, not on the DB.

Here is my suggested response to return in the body:

```JSON
{
   "version": "{{VERSION_OF_SERVICE from the package.json}}",
   "status": {
      "service": "ok", <-- this would only be ok if all the other tests pass
      "database": "ok" <-- this would only be "ok" if the db connectivity test passes
   }
}
```

Let me know what you think.

Reply:
Thank you for the suggestion. 
Decided to stay with the code base solution for mojaloop. Included the health endpoint in the swagger and reporting on the /health end-point.